### ISSUE TITLE
Trajectory extraction

### ISSUE THREAD

**Issue by KrishnanjanP** [mir-group/flare#375]
Hello,

I am newly using flair. So the question might sound naive, but how the trajectory is extracted from the output file after the completion of a otf run ? I tried the script for the otf_parser to do it, but it shows the following issue: ImportError: cannot import name 'otf_parser' from 'flare'. 
Any clue will be helpful.

Thanks,
Krish

**Comment by cjowen1** [comment]
Hi Krish,

If you are using the ACE/SGP version, there should be a *md.xyz file provided that contains the trajectory.

Best,
Cameron

**Comment by KrishnanjanP** [comment]
Hello Cameron,
Thanks for the reply. 
I am using the SGP version, but there are no *md.xyz or *thermo.txt being written. The three file that are being written along with the checkpoint files are *_dft.xyz,  *-hyps.dat,  *.out, and an empty *.xyz file . I have using 'yaml' file as input as described in one of the tutorials.

below are the version details -
python 3.9.18 (main, Sep 11 2023, 13:41:44) 
[GCC 11.2.0]
numpy 1.22.4
ase 3.22.1
flare 1.4.1

Looking forward to your suggestion. Thanks.

best,
Krish

**Comment by cjowen1** [comment]
Has your OTF passed the first DFT calculation? Can you please provide more information, specifically if there are any errors being written?

**Comment by KrishnanjanP** [comment]
It seems that the run is finishing without any error but I noticed that it is calling DFT only once at the beginning. During the rest of the otf the pressure is remaining high. I am running it with NVTBerendsen. I am attaching the MD step vs temperature plot, and the yaml file as text. Thanks again for our time.


[test.txt](https://github.com/mir-group/flare/files/12673569/test.txt)

![T](https://github.com/mir-group/flare/assets/100198808/1a02aa5e-babf-4801-b1d4-43906bec063f)

**Comment by YuuuXie** [comment]
@KrishnanjanP

As @cjowen1 mentioned, we recommend users to use LAMMPS MD for active learning, then you can get the entire trajectory saved in the *_md.xyz, and the DFT frames saved in the *_dft.xyz. You can follow the [tutorial](https://colab.research.google.com/drive/1qgGlfu1BlXQgSrnolS4c4AYeZ-2TaX5Y) which you can also find in the read me page

If you keep using the ASE MD engine, then you could take a look at our unit tests as examples: https://github.com/mir-group/flare/blob/master/tests/test_parse_otf.py, where you can use `OtfAnalysis` to parse information from the .out file

**Comment by KrishnanjanP** [comment]
Hello, 
Thanks for the suggestions. I am currently infact trying with lammps following the tutorial. I will comeback and close this issue once I succesfully run it. 
Thanks again for you time.

Best,
Krish

**Comment by jonpvandermause** [comment]
Closing as stale after the trajectory-extraction guidance in the existing discussion. Please open a new issue with current output files and commands if the documented LAMMPS trajectory path still fails.
